### ISSUE TITLE
Error network request failed when connect

### ISSUE THREAD

**Issue by chuan298** [bluecherrydvr/bluecherry-mobile#1]
Hi, I get error when connect to server:

`TypeError: Network request failed`
`{"line":26270,"column":33,"sourceURL":"http://10.0.2.2:8081/index.bundle?platform=android&dev=true&minify=false"}`

But when I config in bluecherry-apps by changing https for http, it runs.

**Comment by curtishall** [comment]
@chuan298 Are you connecting to a Bluecherry server?  What SSL certificates are you using on that server (generic / snakeoil or letsencrypt?)

**Comment by chuan298** [comment]
@curtishall yes, I am using Bluecherry server and Bluecherry client, It runs normally. My SSL certificate is letsencrypt.

**Comment by tugrul** [comment]
This is not related with main app functionality. This is debug agent connection for dev build app and not necessary https connection for this.

How did you get this error? Can you share your environment? Did you build for yourself from source? Where do you test it (device or emulator)?

**Comment by curtishall** [comment]
@chuan298 Can you provide feedback on this?

**Comment by DanilXaritonov** [comment]
Hello.
I am having the same problem.
We have version 2.8.8 installed.
Is it mandatory to install an SSL certificate?
We have not installed the certificate.
We tried running bild 3 and 4.
https://ibb.co/fkx9yDY

**Comment by curtishall** [comment]
A valid SSL certificate (not a snake oil) certificate is required.  In v3 paid licenses will generate a valid LetsEncrypt certificate for you.  Please remember there is a 50% upgrade fee to upgrade from v2 to v3.

Thanks

**Comment by hudy5** [comment]
I had the same problem, i had v3 server, ssl installed letsencrypt but when i configure connection then i had: Error network request failed :(
